facster.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+
+const readline = require('readline');
+const fs = require('fs')
+const path = require('path')
+const facsterUtils = require('facster-jobs')
+
+const getRepoName = (callback) => {
+    const rl = readline.createInterface({
+      input: fs.createReadStream(path.join(process.cwd(),'.git/config'))
+    });
+  
+    rl.on('line',(line)=>{
+      if(line.includes('url')){
+        const name = line.split('/').slice(-1)[0];
+        callback(name)
+      }    
+    })
```

Comment:
Hmm should we listen for 'close' or 'error' events, this seems kind of fragile..